src/app/Instance/ErrorHandling/ErrorHandlingEdit.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { FormEvent, useCallback, useEffect, useState } from "react";
+import {
+  ActionGroup,
+  Alert,
+  Button,
+  Form,
+  FormAlert,
+  PageSection,
+} from "@patternfly/react-core";
+import { useTranslation } from "@rhoas/app-services-ui-components";
+import ErrorHandlingCreate from "@app/Instance/ErrorHandling/ErrorHandlingCreate";
+
+export interface ErrorHandlingEditProps {
+  apiError?: string;
+  getSchema: (method: string) => Promise<object>;
+  isLoading: boolean;
+  method?: string;
+  onCancelEditing: () => void;
+  onSubmit: (method?: string, parameters?: object, schema?: object) => void;
+  parameters?: Record<string, unknown>;
+  registerValidateParameters: (validationFunction: () => boolean) => void;
+  schema?: object;
+}
+
+export const ErrorHandlingEdit = ({
+  apiError,
+  getSchema,
+  isLoading,
+  method,
+  onCancelEditing,
+  onSubmit: onErrorHandlingSubmit,
+  parameters,
+  registerValidateParameters,
+}: ErrorHandlingEditProps): JSX.Element => {
+  const { t } = useTranslation(["openbridgeTempDictionary"]);
+
+  const [isSubmitted, setIsSubmitted] = useState<boolean>(false);
+  const [errorHandlingMethod, setErrorHandlingMethod] = useState<
+    string | undefined
+  >(method);
+  const [errorHandlingParameters, setErrorHandlingParameters] =
+    useState(parameters);
+
+  const onErrorHandlingParametersChange = useCallback(
+    (method?: string, parameters?: Record<string, unknown>): void => {
+      setErrorHandlingMethod(method);
+      setErrorHandlingParameters(parameters);
+    },
+    []
+  );
+
+  const onSubmit = useCallback(
+    (event: FormEvent<HTMLFormElement>) => {
+      event.preventDefault();
+      onErrorHandlingSubmit(errorHandlingMethod, errorHandlingParameters);
+      setIsSubmitted(true);
+    },
+    [errorHandlingMethod, errorHandlingParameters, onErrorHandlingSubmit]
+  );
+
+  useEffect(() => {
+    if (isSubmitted) {
+      document
+        .querySelector(".pf-m-error")
+        ?.parentElement?.previousElementSibling?.scrollIntoView({
+          behavior: "smooth",
+          block: "start",
+          inline: "nearest",
+        });
+      setIsSubmitted(false);
+    }
+  }, [isSubmitted]);
+
+  return (
+    <Form
+      className="error-handling-edit-form"
+      autoComplete="off"
+      onSubmit={onSubmit}
+    >
+      {apiError && (
```

Comment:
This is an inconsistent state, for sure. What do you think about keeping the save button enabled in case of errors?

Reply:
> This is an inconsistent state, for sure. What do you think about keeping the save button enabled in case of errors?

I agree. Not sure if the same applies when the `cantUpdateInstanceNotActionableState` error happens. In that case probably we should exit the edit mode. wdyt?

Reply:
But then how could you know when you can try again from within the same page? This is similar to the concurrent editing we had to manage for processors

Reply:
We have a disagreement :slightly_smiling_face: 

I see your point in not losing already entered information, but in this scenario I would prioritize preventing users from stepping on each others toes while editing the same thing.

Let's keep your implementation by the way. We can involve UX into the discussion and eventually change anything in a separate ticket.